Source/Blazorise/Components/Table/TableHeaderCell.razor.cs

```diff
@@ -17,19 +17,49 @@ public partial class TableHeaderCell : BaseDraggableComponent
 
     private Cursor cursor;
 
+    private double? fixedLeftPosition;
+
     #endregion
 
     #region Methods
 
+    /// <inheritdoc/>
+    protected override void OnInitialized()
+    {
+        if ( ParentTable is not null )
+        {
+            ParentTable.NotifyTableRowCellInitialized( ParentTableRow, this );
+
+            if ( Fixed )
+            {
+                fixedLeftPosition = ParentTable.GetFixedCellPosition();
+            }
+        }
+
+        base.OnInitialized();
+    }
+
     /// <inheritdoc/>
     protected override void BuildClasses( ClassBuilder builder )
     {
         builder.Append( ClassProvider.TableHeaderCell() );
         builder.Append( ClassProvider.TableHeaderCellCursor( Cursor ) );
+        builder.Append( ClassProvider.TableHeaderCellFixed( Fixed ) );
 
         base.BuildClasses( builder );
     }
 
+    /// <inheritdoc/>
+    protected override void BuildStyles( StyleBuilder builder )
+    {
+        if ( Fixed && fixedLeftPosition != null )
+        {
+            builder.Append( $"left:{fixedLeftPosition:G29}px" );
```

Comment:
what the heck is G29? :D

Reply:
This is to strip the trailing zeros, see https://stackoverflow.com/a/8675368/833106

Reply:
hmm kinda unconventional, I'd go the extra mile and make a string constant with FORMAT_STRIP_TRAILING_ZEROS or something along those lines, we will easily forget this in the future I would say.

Although, to be fair it's easily googleable I guess...